.github/workflows/main.yml

```diff
@@ -173,6 +174,38 @@ jobs:
             OS-${{ runner.os }},
             VM-${{ matrix.platform }},
             Py-${{ steps.python-install.outputs.python-version }}
+            
+  check-changed-folders:
+    name: Check files
```

Comment:
It'd probably be better to be more precise about checking the vendored dirs, not some random arbitrary files.